cirq-core/cirq/qis/measures.py

```diff
@@ -257,7 +259,10 @@ def von_neumann_entropy(
     validate: bool = True,
     atol: float = 1e-7,
 ) -> float:
-    """Calculates the von Neumann entropy of a quantum state in bits.
+    r"""Calculates the von Neumann entropy of a quantum state in bits.
```

Comment:
Optional: This is sometimes specialized to "ebits" (bits of bipartite entanglement).

Reply:
Added to description.